docker-compose.yml

```diff
@@ -102,11 +97,11 @@ services:
       - POSTGRES_PASSWORD=galoy-price-pwd
       - POSTGRES_DB=galoy-price-history
   bitcoind:
-    image: lncm/bitcoind:v23.0
+    image: lncm/bitcoind:v24.0
```

Comment:
maybe is better to do the changes (versions) in this file in a different PR

Reply:
reverted (but thanksfully it's really a no-op)